src/DynamicObjects.jl

```diff
@@ -34,10 +38,6 @@ This dynamic type can be used in function definitions as
 area(what::Rectangle) = what.height * what.width
 ```
 """
-get_sname(name::Symbol) = QuoteNode(name)
```

Comment:
The code was adding the docstring to `get_sname`, not `@dynamic_object`

Reply:
Oh, right!